build/cpp/kvstore/src/kvstore_server.cc

```diff
@@ -1,12 +1,26 @@
 #include "kvstore_server.h"
 
+// key value store server constructor
+kvstore_server::KeyValueStoreServiceImpl::KeyValueStoreServiceImpl(
+    const std::string &store) {
+  LOG(INFO) << "Server constructor commenced";
+  kvstore_ = new kvstore::KVStore(store);
+}
+
+// key value store destructor
+kvstore_server::KeyValueStoreServiceImpl::~KeyValueStoreServiceImpl() {
+  LOG(INFO) << "Server destructor commenced";
```

Comment:
Maybe these logs should be more descriptive, ie, reading from store or writing to store

Reply:
The logs were updated to be more descriptive as suggested.